src/components/organisms/FileTreePane/FileTreePane.tsx

```diff
@@ -309,6 +309,9 @@ const FileTreePane = () => {
   }, [selectedPath]);
 
   const onSelect = (selectedKeysValue: React.Key[], info: any) => {
+    if (!['.yaml', '.yml'].includes(path.extname(info.node.key))) {
```

Comment:
you could check against the appConfig.fileIncludes array?

Reply:
but then the user could include other file types that can be selected and shown in the editor, do we want that for now?

Reply:
those files would be parsed for manifests - so I think it's ok to make them editable also

Reply:
I see, okay, will modify it